py3status/modules/wifi.py

```diff
@@ -45,11 +45,9 @@
 @author Markus Weimar <mail@markusweimar.de>
 @license BSD
 """
-
```

Comment:
I think I might add something like

```__Note: Some distributions eg Debian require `wi` to be run with privileges.  In this case you will need to use the `use_sudo` configuration parameter.__```

to the docstring after the `Requires:` section.